### DISCUSSION TITLE
Fleet Dashboard "Agent Health" links 404 page not found

### DISCUSSION THREAD

**Discussion by V1p3rACE** [Security-Onion-Solutions/securityonion#11017] [2.4] [answered]
HI,
Firstly Security Onion is awesome - great work! I Just want to see if I was the only one having this issue - When going into the Elastic Fleet manager and clicking on an agent to click into the "View more agent metrics" - The links on the left within the "Agent Health" section such as "Overview", "Agent Info", "Agent Metrics" or "Integrations" will lead to a "404 page not found error" . It seems that the url is missing "kibana" before /app/dashboards#/ - If I put this in manually before the app then I can view the links fine.  Not sure its just my install but I have tried the previous RC the issue is there also? I'm currently using 2.4 rc2

**Comment by defensivedepth** [comment, marked as answer]
I have created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/11018
